airbyte-proxy/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Inspired by https://medium.com/pernod-ricard-tech/adding-basic-authentication-with-nginx-as-a-reverse-proxy-a229f9d12b73
+
+FROM nginx:latest
+
+ARG VERSION=0.40.12
+
+ENV APPLICATION airbyte-proxy
+ENV VERSION ${VERSION}
+
+RUN apt-get update -y && apt-get install -y apache2-utils && rm -rf /var/lib/apt/lists/*
+
+# Basic auth credentials
+ENV BASIC_AUTH_USERNAME ""
+ENV BASIC_AUTH_PASSWORD ""
```

Comment:
one thought: 
* should we remove these envs? Just reading the docker file it looks like these are not set today, when actually these are injected via the .env file.
* or is this how we are injecting the default value today? Should we leave a comment here explaining that in most cases these should be overriden?

Reply:
I'll remove them!